File: dotnet/Atlas.Averbacao/Services/Contratacao/ContratacaoService.cs
using Atlas.Averbacao.Api.Contratacao;

namespace Atlas.Averbacao.Services.Contratacao;

/**
 * O servidor escolheu uma oferta sua. Use request.OfertaId e
 * request.CorrelacaoId para reservar/confirmar no seu core.
 *
 * Se você não precisa desse aviso, deixe o método como está.
 */
public class ContratacaoService : IContratacaoService
{
    public Task Iniciada(ContratacaoRequest request)
    {
        // await suaApi.ConfirmarProposta(request.OfertaId);
        _ = request;
        return Task.CompletedTask;
    }
}

File: java/src/main/java/br/com/atlas/averbacao/services/contratacao/ContratacaoService.java
package br.com.atlas.averbacao.services.contratacao;

import br.com.atlas.averbacao.api.contratacao.ContratacaoRequest;
import org.springframework.stereotype.Service;

/**
 * O servidor escolheu uma oferta sua. Use request.ofertaId() e
 * request.correlacaoId() para reservar/confirmar no seu core.
 *
 * Se você não precisa desse aviso, deixe o método como está.
 */
@Service
public class ContratacaoService implements ContratacaoContract {
    @Override
    public void iniciada(ContratacaoRequest request) {
        // suaApi.confirmarProposta(request.ofertaId());
    }
}
